Andy Solar System Test/Assets/FilterScript.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FilterScript : MonoBehaviour {
+
+	// Use this for initialization
+	void Start () {
+		
+	}
+	
+	// Update is called once per frame
+	void Update () {
+		
+	}
+    public void filterBy(string crit)
+    {
+        switch (crit)
+        {
+            case "alpha":
+
+                break;
+            case "sun":
+
+                break;
+
+            case "near":
+
+                break;
+            case "most":
```

Comment:
Not sure what this is doing, but we have 5 sortable options, not 4.

1. THE_UNIVERSE.nearestEarth();
2. THE_UNIVERSE.mostPlanets();
3. THE_UNIVERSE.mostLikelyHabitable();
4. THE_UNIVERSE.mostLikeSun();
5. THE_UNIVERSE.sortedAlphabetical();

Reply:
Can you please fix it? 
I looked into the code and this is what I saw

Reply:
I can't find the unity object that's calling this code.  Also, I don't want to commit changes to a .unity file if you're working on it as well since that'll cause conflicts.